src/__tests__/fixtures/spectral-lf.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
This is a bit weird case.
Usually, our YAML files start with actual nodes... but not on this occasion.
I decided to report 0 line and 0 character if a given path cannot be found in a document.
Reporting actual character does not make much sense since there is no node there anyway.
Ideally, we would report the second line, as this is where the actual document starts.

@marbemac thoughts? Going with the latter idea will require a few changes, but it's doable.

Reply:
How is `---` represented in the AST? What happens in the following situation - what line is Foo reported as?

```
---
---

Foo
```